packages/core/test/hardhat/data-verification-mechanism/FixedSlashSlashingLibrary.js

```diff
@@ -0,0 +1,31 @@
+const { assert } = require("chai");
+const hre = require("hardhat");
+const { didContractThrow } = require("@uma/common");
+
+const { getContract, web3 } = hre;
+const { toWei } = web3.utils;
+
+const SlashingLibrary = getContract("FixedSlashSlashingLibrary");
+
+const baseSlashAmount = toWei("0.0016");
+const governanceSlashAmount = toWei("0");
+
+describe("FixedSlashSlashingLibrary", function () {
+  let accounts, owner;
+  beforeEach(async function () {
+    accounts = await web3.eth.getAccounts();
+    [owner] = accounts;
+  });
+  it("Validate baseSlashAmount", async function () {
+    const invalidBaseSlashAmount = toWei("1");
+    assert(
+      await didContractThrow(SlashingLibrary.new(invalidBaseSlashAmount, governanceSlashAmount).send({ from: owner }))
```

Comment:
Nit: In general, I think it's better to specify the failing message even if we don't do it in the rest of tests

```suggestion
      await didContractThrow(SlashingLibrary.new(invalidBaseSlashAmount, governanceSlashAmount).send({ from: owner }), "Invalid base slash amount")
```

Reply:
agree, created `didContractRevertWith` to match exact rever messages